pkg/plugins/ansible/v1/init.go

```diff
@@ -226,29 +218,11 @@ func addInitCustomizations(projectName string, componentConfig bool) error {
 	// replace the default ports because ansible has been using another one
 	// todo: remove it when we be able to change the port for the default one
 	// issue: https://github.com/operator-framework/ansible-operator-plugins/issues/4331
-	err = util.ReplaceInFile(managerFile, "port: 8081", "port: 6789")
+	err = util.ReplaceInFile(managerFile, "8081", "6789")
 	if err != nil {
 		return err
 	}
 
-	if componentConfig {
-		managerConfigFile := filepath.Join("config", "manager", "controller_manager_config.yaml")
-		err = util.ReplaceInFile(managerConfigFile, "8081", "6789")
-		if err != nil {
-			return err
-		}
-		// Remove the webhook option for the componentConfig since webhooks are not supported by ansible
-		err = util.ReplaceInFile(managerConfigFile, "webhook:\n  port: 9443", "")
-		if err != nil {
-			return err
-		}
-	} else {
-		err = util.ReplaceInFile(managerProxyPatchFile, "8081", "6789")
```

Comment:
Does this need to stay but be changed to `managerMetricsPatchFile`?

Reply:
My understanding is this is intentional with the manager config file being discontinued along with kube-rbac-proxy. @joelanford can you confirm?